code/go/0chain.net/smartcontract/zcnsc/nodes_reader.go

```diff
@@ -26,11 +26,7 @@ func GetAuthorizerNode(id string, ctx state.StateContextI) (*AuthorizerNode, err
 func GetUserNode(id string, ctx state.StateContextI) (*UserNode, error) {
 	node := NewUserNode(id, 0)
 	err := ctx.GetTrieNode(node.GetKey(), node)
-	if err != nil {
-		return nil, err
-	}
-
-	return node, nil
+	return node, err
```

Comment:
we will break mint probably, Imagine user transfers wZCN to 0chain and has no account here, we should create new account for such user and mint him tockens if all is ok

Reply:
Impossible case because a client cannot call the smart contract function without having inititialized the wallet and started working with GOSDK.

Reply:
Wallet initialization do not create any records in MPT, we create read/write pools when wallet is initialized, but I'm not sure that igt creates UserNode, we better handle this situation.